api/routes/forgot-password/controllers.js

```diff
@@ -0,0 +1,74 @@
+const BaseController = require('base/controllers/baseController');
+const DB = require('models');
+const config = require('config');
+const ApiError = require('base/error');
+
+class ForgetPasswordController extends BaseController {
+  async post() {
+    const { email } = this.request.body;
```

Comment:
we can get the user from `this.request.user`

Reply:
i think user won't be logged in this flow

Reply:
sorry yes !